pkg/oc/cli/admin/policy/modify_scc.go

```diff
@@ -3,19 +3,18 @@ package policy
 import (
 	"errors"
 	"fmt"
-	"io"
 
 	"github.com/spf13/cobra"
 
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/kubernetes/pkg/kubectl/cmd/templates"
 	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
 	"k8s.io/kubernetes/pkg/kubectl/genericclioptions"
+	"k8s.io/kubernetes/pkg/kubectl/genericclioptions/printers"
+	"k8s.io/kubernetes/pkg/kubectl/scheme"
 
-	securityclient "github.com/openshift/client-go/security/clientset/versioned"
-	securitytypedclient "github.com/openshift/client-go/security/clientset/versioned/typed/security/v1"
+	securityv1client "github.com/openshift/client-go/security/clientset/versioned/typed/security/v1"
```

Comment:
`securityv1typedclient`

Reply:
:/ was hoping to leave these convention changes for a later PR (per my comment here https://github.com/openshift/origin/pull/20810#discussion_r214444989)